cli/script/command-parser.ts

```diff
@@ -275,10 +276,28 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
             .example("logout", "Log out and end your session");
         addCommonConfiguration(yargs);
     })
+    .command("patch", "Update the metadata for an existing release", (yargs: yargs.Argv) => {
+        yargs.usage(USAGE_PREFIX + " patch <appName> <deploymentName> [--label <label>] [--description <description>] [--mandatory] [--rollout <rolloutPercentage>]")
+            .demand(/*count*/ 3, /*max*/ 3)  // Require exactly three non-option arguments.
+            .example("patch MyApp Production --des \"Updated description\" -r 50", "Update the description of latest release for \"MyApp\" app's \"Production\" deployment and update rollout value to 50")
+            .example("patch MyApp Production -l v3 --des \"Updated description for v3\"", "Update the description of the release with label v3 for \"MyApp\" app's \"Production\" deployment")
+            .option("label", { alias: "l", default: null, demand: false, description: "The label of the release to be updated, defaults to the latest release", type: "string" })
```

Comment:
Either:

    ...updated, which defaults to...

...or:

    ...updated (default:  the latest release)